cli/cmd/docs/docs.go

```diff
@@ -0,0 +1,36 @@
+package docs
+
+import (
+	"log"
+
+	"github.com/rilldata/rill/cli/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+// docsCmd represents the docs command
+func DocsCmd(docsUrl string) *cobra.Command {
```

Comment:
Just hard-code the docs url as a const at the top of this file instead of the indirection here. It will never need to be configured.

Reply:
What if it change? will update it in the code?

Reply:
Yeah